deploy/helm/postee/values.yaml

```
@@ -186,7 +186,7 @@ posteUi:
     # usernameKey: username
     # passwordKey: password
   image: aquasec/postee-ui
-  tag: "2.12.0-amd64"
+  # tag: "2.12.0-amd64"
```

I think this comment might be confusing.
Can we add comment about this line?
Something like this:
```suggestion
  # By default `tag` is taken from `.Chart.AppVersion`
  # To use different version - uncomment this line and enter the desired version
  # tag: ""
```